### ISSUE TITLE
[Feature Request] More bulk actions (add to playlist, change channel, etc)

### ISSUE THREAD

**Issue by Skehmatics** [Chocobozzz/PeerTube#1980]
On the /my-account/videos page, multi-select is allowed for the listed videos, but the only bulk action available is "Delete." For users with many videos, organizational actions such as adding to a playlist or changing the host channel would be very useful to have here as well.

**Comment by achaiah** [comment]
Hi, is this still on the roadmap? It is very difficult to organize a large library when you can't bulk-edit any videos. How are we supposed to add them to playlists, change tags, move them between channels etc?  One at a time is simply not feasible for hundreds or thousands of videos!

**Comment by rdnaskelz** [comment]
Also interested in bulk editing - not sure why but after uploading about a 100 videos "publish after transcoding" was not active (maybe I missed some setting but I did publish it using API and set the privacy setting to "1") so publishing them one by one is not an easy task. I'm working my way through API but would be grateful to be able to do it within the UI

**Comment by tilllt** [comment]
Yes, another usage scenario: we have co-workers that added hundreds of video but now are leaving our organization. Bulk editing ownership of Videos & plaxlists,  transferring ownership of Channels, bulk adjusting privacy settings etc is totally essential to efficiently administrate a peertube instance. Can this be scripted as a workaround in the meanwhile?

**Comment by candideu** [comment]
My use case: I've imported videos from a channel on another instance, and they got added to a channel from the old channel. I want to move all those newly imported videos into existing channels, but there's no way to do so...

**Comment by candidexmedia** [comment]
My use case: I've imported videos from a channel on another instance, and they got added to a channel from the old channel. I want to move all those newly imported videos into existing channels, but there's no way to do so...

**Comment by MarkWieczorek** [comment]
For me, the mail bulk action features that I would like are:
* change video language
* add to playlist

**Comment by Chocobozzz** [comment]
Implemented in https://github.com/Chocobozzz/PeerTube/commit/b7542c17b6b9dadeee1c58f36002da1c027e98d5
